streaming/base/converters/dataframe_to_mds.py

```diff
@@ -54,47 +54,41 @@
 }
 
 
-def is_iterable(obj: object) -> bool:
```

Comment:
we don't need this method anymore?

Reply:
one-line method used once, also

issubclass(type(obj), Iterable) -> isinstance(obj, Iterable)